src/Components/IndexScene/ToolbarScene.tsx

```diff
@@ -0,0 +1,144 @@
+import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import { Dropdown, Switch, ToolbarButton, useStyles2 } from '@grafana/ui';
+import React from 'react';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+import { config } from '@grafana/runtime';
+import { reportAppInteraction, USER_EVENTS_ACTIONS, USER_EVENTS_PAGES } from '../../services/analytics';
+import { AGGREGATED_METRIC_START_DATE } from '../ServiceSelectionScene/ServiceSelectionScene';
+import pluginJson from '../../plugin.json';
+const AGGREGATED_METRICS_USER_OVERRIDE_LOCALSTORAGE_KEY = `${pluginJson.id}.serviceSelection.aggregatedMetrics`;
+
+export interface ToolbarSceneState extends SceneObjectState {
+  isOpen: boolean;
+  options: {
+    aggregatedMetrics: {
+      active: boolean;
+      userOverride: boolean;
+      disabled: boolean;
+    };
+  };
+}
+export class ToolbarScene extends SceneObjectBase<ToolbarSceneState> {
+  constructor(state: Partial<ToolbarSceneState>) {
+    const overrideFalsy =
+      localStorage.getItem(AGGREGATED_METRICS_USER_OVERRIDE_LOCALSTORAGE_KEY) === 'false' ? false : undefined;
+    const userOverride =
+      localStorage.getItem(AGGREGATED_METRICS_USER_OVERRIDE_LOCALSTORAGE_KEY) === 'true' ? true : overrideFalsy;
```

Comment:
Not sure I understand this. What stops us from using 
```
const userOverride = localStorage.getItem(AGGREGATED_METRICS_USER_OVERRIDE_LOCALSTORAGE_KEY) === 'true' ? true : false;
```

Reply:
in local storage we can have 'false' || 'true' || null, but I agree this was written confusedly. I cleaned it up a bit and everything appears to still be working. Make sure to test queries that start before aug 30, and with empty local storage state.